operations/services.md

```diff
@@ -186,9 +191,15 @@ Returns all availability blocks filtered by services, unique identifiers and oth
 | `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were created. |
 | `UpdatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were updated. |
 | `CollidingUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s are active. |
-| `ExternalIdentifiers` | string | optional, max 1000 items | Identifiers of [Availability block](#availability-block)s from external systems |
+| `States` | array of string [Availability block state](#availability-block-state) | optional | States the availability blocks should be in. |
```

Comment:
if we're getting rid of the fallback, then should the `States` parameter be `required` rather than `optional`?

Reply:
The current behavior of the API does not require the state so I'd not specify it as required. If the state is not provided, we disregard the filter completely.